MVC/errors.js

```diff
@@ -1,16 +1,18 @@
 exports.handleCustomErrors = (err, req, res, next) => {
-    if (err.status && err.msg) {
-      res.status(err.status).send({ msg: err.msg });
-    } else next(err);
-  };
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
 
-  exports.handlePsqlErrors = (err, req, res, next) => {
-    if (err.code === '22P02') {
-      res.status(400).send({ msg: 'Bad request' });
-    } else next(err);
-  };
+exports.handlePsqlErrors = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Bad request" });
+  } else if (err.code === "23503") {
+    res.status(400).send({ msg: "Bad request" });
```

Comment:
seeing as these two have the same outcome - could you double up the first if statement?